jest/setup.js

```diff
@@ -105,6 +105,12 @@ const mockNativeModules = {
   FbRelayNativeAdapter: {
     updateCLC: jest.fn(),
   },
+  Geolocation: {
+    getCurrentPosition: jest.fn(),
+    watchPosition: jest.fn(),
+    clearWatch: jest.fn(),
```

Comment:
Ah yes, my bad. I guess the mocks you have on Geolocation should move to LocationObserver then